LogMaker4GoogleDocs/logmaker_4_googledocs/docgen.py

```diff
@@ -824,6 +857,32 @@ def insertImageToExperimentLog(documentID, scanNumber, row, column, image_path):
     # but it's really just a temporary location for the image before 
     # moving into the relevant experiment log. No real need to specify
     # other directories for other experiments
+    
+    experiment_mapping = {'Undulator':'HTUparameters.ini'}
+    
+    config_file = experiment_mapping.get(experiment, None)
+    if config_file:
+        # Create the ConfigParser object
+        experiment_config = configparser.ConfigParser()
+
+        # Get the directory of the current script (docgen.py)
+        experiment_config_dir = Path(__file__).parent
+
+        # Construct the path to config.ini
+        config_path = experiment_config_dir / config_file
+
+        # Load the configuration file
+        experiment_config.read(config_path)
+
+        # Debugging: Verify the loaded sections
+        if not experiment_config.sections():
+            print(f"Failed to load config file from: {config_path}")
+        else:
+            print(f"Successfully loaded config file: {config_path}")
+            
+        documentID = experiment_config['DEFAULT']['logid']  
+    
+    
     image_id = uploadImage(image_path, '1O5JCAz3XF0h_spw-6kvFQOMgJHwJEvP2')
```

Comment:
Is this just a random folder in the google drive that images are uploaded to before being transferred to the gdoc?  Maybe a comment on where exactly this is could be handy.

Also, while we do have unlimited storage, do we want to consider removing these files once they are uploaded?